src/interfaces/packagingInterfacesAndType.ts

```diff
@@ -223,3 +223,16 @@ export type PackageVersionReportResult = Partial<PackagingSObjects.Package2Versi
 export type PackageVersionCreateReportProgress = PackageVersionCreateRequestResult & {
   remainingWaitTime: Duration;
 };
+
+export type Package1VersionCreateRequest = Pick<
+  PackagingSObjects.PackageUploadRequest,
+  | 'MetadataPackageId'
+  | 'VersionName'
+  | 'Description'
+  | 'MajorVersion'
+  | 'MinorVersion'
+  | 'IsReleaseVersion'
+  | 'ReleaseNotesUrl'
+  | 'PostInstallUrl'
+  | 'Password'
+>;
```

Comment:
Are all of these fields required to create a package1 version?  I suspect not.  I think you'll have to do an intersection of the required fields and optional fields.  something like:
`Partial<Pick<PackagingSObjects.PackageUploadRequest, {optional_fields}>> & Pick<PackagingSObjects.PackageUploadRequest, {required_fields}>`